pkg/api/handlers/compat/images_build.go

```diff
@@ -790,7 +790,7 @@ func extractTarFile(r *http.Request) (string, error) {
 	}
 
 	buildDir := filepath.Join(anchorDir, "build")
-	err = os.Mkdir(buildDir, 0700)
+	err = os.Mkdir(buildDir, 0o700)
```

Comment:
I know this kind of notation is happy in python, is it good in Go too?

Reply:
@TomSweeneyRedHat yes, gofumpt makes the change automatically as well as other clean-ups.